client/components/credit-card-form-fields/index.jsx

```diff
@@ -144,7 +117,7 @@ function CreditCardNumberField( { translate, createField, getErrorMessage, card
 	return createField( 'number', CreditCardNumberInput, {
 		inputMode: 'numeric',
 		label: cardNumberLabel,
-		placeholder: '•••• •••• •••• ••••',
+		placeholder: disabled ? ' ' : '1234 1234 1234 1234',
```

Comment:
Nice. Although now I can't use those dots to tell which version of the credit card field I'm seeing... oh well.

Reply:
The reason I took them out was because they would disappear and then it would take another second before the numbers loaded in. I was going for something a little more seamless in the loading experience.